code/modules/mob/living/simple_animal/abnormality/_tools/zayin/touch.dm

```diff
@@ -2,28 +2,60 @@
 	name = "don't touch me"
 	desc = "You feel like you shouldn't touch this."
 	icon_state = "touch"
+	var/cooldown
 	var/list/bastards = list()
+	var/list/breaching_bastards = list()
+
+/obj/structure/toolabnormality/touch/examine(mob/user)
+	. = ..()
+	. += "<span class='info'>Pressing it while on help intent will breach all abnormalities instead of ending the shift.</span>"
 
 //I had to.
 /obj/structure/toolabnormality/touch/attack_hand(mob/living/carbon/human/user)
-	if(user.ckey in bastards)
+	if(cooldown > world.time)
+		to_chat(user, "<span class='notice'>THE BUTTON CANNOT BE PRESSED RIGHT NOW.</span>")
+		return
+
+	var/round_end = (user.a_intent != INTENT_HELP)
+	if((user.ckey in bastards) || (!round_end && (user.ckey in breaching_bastards)))
 		to_chat(user, "<span class='userdanger'>THE BUTTON REJECTS YOU.</span>")
-		user.gib()
```

Comment:
Honestly keep the gib as punishment people have to learn

Reply:
There is absolutely no reason to kill someone who simply tried to press it again.